src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1383,13 +1401,22 @@ MavlinkReceiver::handle_message_set_attitude_target(mavlink_message_t *msg)
 
 				/* Publish attitude rate setpoint if bodyrate and thrust ignore bits are not set */
 				///XXX add support for ignoring individual axes
```

Comment:
Comment now unnecessary

Reply:
Not sure if unnecessary. I implemented Yawrate control, not for the other axis though. Still WIP